### DISCUSSION TITLE
How to optimize for large (over 2M) rows in a table ?

### DISCUSSION THREAD

**Discussion by paramwork** [rappasoft/laravel-livewire-tables#1008] [Q&A]
My table is not responsive and takes forever to load. The `count` is using too many resources.

Can you please suggest ways to optimize it?

**Comment by rappasoft** [comment]
Try changing the pagination method to simple: https://rappasoft.com/docs/laravel-livewire-tables/v2/pagination/available-methods#content-setpaginationmethod

**Reply by paramwork** [reply]
Thanks for the reply. 

Is there a similar method available in v1?
